mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -81,6 +81,13 @@ private[clustering] trait KMeansParams extends Params with HasMaxIter with HasFe
   def getInitSteps: Int = $(initSteps)
 
   /**
+   * Param for KMeansModel to use for warm start.
```

Comment:
Actually, we also need to be clear that the `initMode` is entirely ignored when setting an initial model. Let's add it to the doc of the `initMode` param